specification/iotsecurity/resource-manager/readme.md

```diff
@@ -36,6 +36,10 @@ directive:
     from: onPremiseSensors.json
     where: '$.paths["/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/onPremiseSensors"].get'
     reason: The list returns limited number of items
+  - suppress: PageableOperation
+    from: vulnerabilities.json
+    where: '$.paths["/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/locations/{iotDefenderLocation}/deviceGroups/{deviceGroupName}/devices/{deviceId}/vulnerabilities"].get'
+    reason: The list returns limited number of items
```

Comment:
Looks you do support pagination in yout collection call. WHy do you need this ?

Reply:
mistake